test/action.spec.js

```diff
@@ -0,0 +1,112 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const mock = require('mock-require');
+const log = require('npmlog');
+const path = require('path');
+
+const action = (config, args) => require('../src/action')(config, args || []);
+
+log.level = 'silent';
+
+describe('action', () => {
+
+  beforeEach(() => {
+    delete require.cache[require.resolve('../src/action')];
```

Comment:
Since require puts all the things into `cache`, we have to invalidate it before each suite runs so that mocks are working. In `rnpm`, it works w/o that because we require inside functions (during runtime). In this case, we have top-level requires for e.g. `registerNativeModuleIOS` and that had to be handled. We can move that to `utils` once we find it reusable.